File: ems_interface/src/main/java/com/ems/service/SectionService.java
package com.ems.service;

import com.ems.pojo.Section;

import java.util.List;
import java.util.Map;

/**
 * @InterfaceName: SectionService
 * @Description: TODO
 * @Autor:13528
 * @Date: 2020/3/15 16:43
 * @Version 1.0
 **/
public interface SectionService {
    //查询全部部门信息
    public List<Section> findAll();
    //查询单个部门信息
    public Section findOne(Integer id);
    //更改部门信息
    public void updateSection(Section section);
    //新建部门信息
    public void addSection(Section section);
    //删除部门信息
    public void deleteSection(Integer[] ids);
    //下拉框数据
    List<Map> selectSectionList();
}

File: ems_hr_web/src/main/webapp/js/service/sectionService.js
app.service('sectionService',function ($http) {

    //查询全部部门信息
    this.findAll=function () {
        return $http.get('../section/findAll.do');
    };

    //查询单个部门信息
    this.findOne=function (id) {
        return $http.get('../section/findOne.do?id='+id);
    };
    //修改部门信息
    this.updateSec=function (section) {
        return $http.post('../section/updateSection.do',section);
    };
    //新建部门信息
    this.addSec=function (section) {
        return $http.post('../section/addSection.do',section);
    };
    //删除部门信息
    this.dele=function (ids) {
        return $http.get('../section/dele.do?ids='+ids);
    };

    //下拉列表数据
    this.selectSectionList=function(){
        return $http.get('../section/selectSectionList.do');
    }

});